### PR TITLE
Add `kyoto` page to `Syncing`

### LINKED ISSUE #87: Add Page: Kyoto Compact Block Filter Client
A page on Kyoto would be awesome!

### PR DESCRIPTION
**Pull request by rustaceanrob**
Closes #87

### CODE REVIEW

File: docs/cookbook/syncing/kyoto.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+```toml
+bdk_kyoto = { version = "0.4.0", default-features = false, features = ["rusqlite", "wallet", "callbacks"] }
+bdk_wallet = { version = "1.0.0-beta.5" }
+tokio = { version = "1.37", features = ["full"], default-features = false }
+```
+
+### Create and sync the wallet
+
+```rust
```
Comment (riverKanies) [inline docs/cookbook/syncing/kyoto.md:23]: looks like this is the same code as in the actual code file correct? if so then you can pull the code using a code snippet (see other pages in book like quickstart) @rustaceanrob

**Comment by rustaceanrob** [comment]
Updated to:

1. Use links to example code in markdown
2. Mention mempool tradeoffs

**Comment by thunderbiscuit** [comment]
@rustaceanrob would you mind rebasing to pick up the commits from valuedmammal? This looks ready to go and I'll merge so we have it for the 1.0 release.

**Comment by rustaceanrob** [comment]
I just happened to be looking at this PR. Rebased.

**Pull request merged**